app/assets/v2/js/pages/townsquare.js

```diff
@@ -1,5 +1,36 @@
 $(document).ready(function() {
 
+  let resizeTimeout = null;
+
+  const collapseAccordions = function() {
+    const window_width = $('body').width();
+
+    $('.townsquare_block-header').each(function(e) {
+      const target_id = $(this).data('target');
+      const item = localStorage.getItem(target_id.replace(/^#/, ''));
+
+      if (window_width <= 992) {
+        if (item && item !== 'false' && !$(this).hasClass('collapsed')) {
+          $(this).trigger('click', true);
+        }
+      } else if (item && item !== 'false' && $(this).hasClass('collapsed')) {
+        $(this).trigger('click', true);
+      }
+    });
+    if (window_width > 768) {
+      $('#mobile_nav_toggle li a').removeClass('active');
+      $('.feed_container,.actions_container').removeClass('hidden');
```

Comment:
would 992 and 768 be reused in other places ? if so, would it make sense to make them to const variables and throw them in a common place ?

Reply:
I've abstracted these to `shared.js` bce95beb6f0bd358b1cef028a0cbd0ac806d48e8